### PR TITLE
setup: Automatically setup submodules if missing

### PR DESCRIPTION
**Pull request by pettermk**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -10,6 +10,10 @@ if [ $EUID -ne 0 ]; then
   exit 1
 fi
 
+if git submodule status --recursive | grep -q "^-"; then
+  sudo -u $SUDO_USER git submodule update --init
```
Comment (maliberty) [inline setup.sh]: Why are you using sudo here?  It shouldn't be necessary.  Only the installer needs sudo.

Comment (pettermk) [reply setup.sh]: Makes sense - I've removed the sudo statement

File: setup.sh [hunk 2]
```diff
@@ -10,6 +10,10 @@ if [ $EUID -ne 0 ]; then
   exit 1
 fi
 
+if git submodule status --recursive | grep -q "^-"; then
+  git submodule update --init
```
Comment (maliberty) [inline setup.sh]: Please add --recursive

Comment (pettermk) [reply setup.sh]: Done

**Review by maliberty — APPROVED** [review]

**Pull request merged**